src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  --color-col: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
```

Comment:
I don't think that you need a vertical padding, also horizontal one should be 50px, not 20